docs/reference/index-modules.asciidoc

```diff
@@ -47,8 +47,10 @@ NOTE: The number of shards are limited to `1024` per index. This limitation is a
 `index.number_of_routing_shards`::
 +
 ====
-Number of routing shards used to <<indices-split-index,split>> an index.
+Integer value used with <<index-number-of-shards,`index.number_of_shards`>> to
+route documents to a primary shard. See <<mapping-routing-field>>.
 
+{es} also uses this value when <<indices-split-index,splitting>> an index.
```

Comment:
Small comment, maybe we can remove the 'also' to strengthen this. My understanding is that the whole reason we have `index.number_of_routing_shards` is to make general splits/ shrinks possible, it doesn't really have a motivation on its own.

Reply:
Good feedback. Thanks!